packages/material-ui-icons/src/AirplayTwoTone.js

```diff
@@ -2,5 +2,5 @@ import React from 'react';
 import createSvgIcon from './utils/createSvgIcon';
 
 export default createSvgIcon(
-  <g><g><path d="M6 22h12l-6-6z" /><path d="M21 3H3c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h4v-2H3V5h18v12h-4v2h4c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z" /></g>
+  <><g fill="none"></g><path d="M6 22h12l-6-6z" /><path d="M21 3H3c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h4v-2H3V5h18v12h-4v2h4c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z" /></>
```

Comment:
Looks wrong 🙂

Reply:
It doesn't need to be explicit.

If it was just one file, then sure, but thousands?

Reply:
@mbrookes thank you for pointing it out 🙂